tvloader/parser2v1/parser.go

```diff
@@ -20,6 +20,12 @@ func ParseTagValues(tvs []reader.TagValuePair) (*spdx.Document2_1, error) {
 			return nil, err
 		}
 	}
+	if parser.file != nil && parser.file.FileSPDXIdentifier == nullSpdxElementId2_1 {
```

Comment:
Same comments here and for the next few lines, tell the user which file name / Package name is causing the problem.